spec/lib/dfid-transition/transform/unescape_html_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'dfid-transition/transform/unescape_html'
+
+module DfidTransition::Transform
+  describe UnescapeHtml do
+    subject(:unescape_html) do
+      DfidTransition::Transform::UnescapeHtml.run(string_input)
+    end
+
+    let(:string_input) { 'a normal string' }
+
+    context 'when given single html escaped HTML' do
+      let(:string_input) { '&lt;p&gt;hello&lt;/p&gt;' }
+
+      it "returns an unescaped string" do
+        expect(unescape_html).to eql('<p>hello</p>')
+      end
+    end
+
+    context 'it has double-escaped HTML' do
```

Comment:
We unescape three times in the source – is there a triple-escaped example in here?